src/common/Footer.jsx

```diff
@@ -1,7 +1,119 @@
 import React from 'react';
+import { Box, Grid, Typography, Link, makeStyles } from '@material-ui/core';
 
-const Footer = () => {
-    return (<h1>This is Footer.</h1>)
-}
+const useStyles = makeStyles((theme) => ({
+    root: {
+      flexGrow: 1,
+      paddingTop: theme.spacing(4),
+      paddingRight: theme.spacing(2),
+      paddingBottom: theme.spacing(10), 
+      paddingLeft: theme.spacing(2),
+      textAlign: 'center',
+      backgroundColor: '#34495E', 
+      color: '#FFFFFF', 
+    },
+    title: {
+        fontweight: 400, 
+    }, 
+    listOfItems: {
+        display: 'flex',
+        flexDirection: 'column', 
+        
+    }, 
+    links: {
+      color: '#C2C8CE',
+    }
+  }));
+
+
+  function Footer() {
+    const classes = useStyles();
+    const preventDefault = (event) => event.preventDefault();
+
+    return (
+      <div className={classes.root}>
+        <Box>
+
+        
+        <Grid container spacing={3}>
+         
+          <Grid item xs={12} sm={6} md={3}>
+            <Typography variant="h5" className={classes.title}>Jobs By Location</Typography>
+            <div className={classes.listOfItems}>
+                {
+                    ['bangalore', 'gurgaon', 'hyderabad', 'mumbai'].map((location, i) => (
+                            <Link 
+                                key={i} 
+                                href="#" 
+                                onClick={preventDefault}
+                                className={classes.links}
+                            >
+                                {`Jobs in ${location[0].toUpperCase() + location.slice(1, location.length)}`}
+                            </Link>
+                        ))
+                }
+
+            </div>
+            
+          </Grid>
+          <Grid item xs={12} sm={6} md={3}>
+            <Typography variant="h5" className={classes.title}>Jobs By Function</Typography>
+            <div className={classes.listOfItems}>
+                {
+                    ['Frontend', 'Backend', 'FullStack', 'Cloud'].map((role, i) => (
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
                    ["Frontend", 'Backend', 'FullStack', 'Cloud'].map((role, i) => (
```